aws-glue/jobs/reporting-ratio_stats.py

```diff
@@ -110,329 +121,314 @@ def run_query_get_result(
-    athena_db,
-    s3_output,
-    s3_prefix
+    athena_client, s3_bucket, SQL_QUERY, athena_db, s3_output, s3_prefix
 )
 
 # Retrieve s3 location of Athena query result and retrieve it
-target = 's3://'+ s3_bucket + '/' + r_file_object.key
+target = "s3://" + s3_bucket + "/" + r_file_object.key
 
 pull = pd.read_csv(target)
 pull = pull[pull.ratio > 0 & pull.ratio.notnull()]
 
 # Delete all query results for this job from s3 bucket
-response = s3_client.list_objects_v2(Bucket = s3_bucket, Prefix = s3_prefix)
-
-for object in response['Contents']:
-    if re.search("csv", object['Key']):
-        print('Deleting', object['Key'])
-        s3_client.delete_object(Bucket = s3_bucket, Key = object['Key'])
-
-
-# COD, PRD, PRB functions ----
-def cod(ratio):
-
-    n = ratio.size
-    median_ratio = ratio.median()
-    cod = 100 / median_ratio * (sum(abs(ratio - median_ratio)) / n)
-
-    return cod
-
-def prd(fmv, sale_price):
-
-    ratio = fmv / sale_price
-    prd = ratio.mean() / np.average(a = ratio, weights = sale_price)
-
-    return prd
-
-def prb(fmv, sale_price):
-
-    ratio = fmv / sale_price
-    median_ratio = ratio.median()
-
-    lhs = (ratio - median_ratio) /median_ratio
-    rhs = np.log(((fmv / median_ratio) + sale_price) / 2) / np.log(2)
-
-    lhs = np.array(lhs)
-    rhs = np.array(rhs)
-
-    return sm.OLS(lhs, rhs).fit()
+response = s3_client.list_objects_v2(Bucket=s3_bucket, Prefix=s3_prefix)
 
-# Functions to determine whether assessment fairness criteria has been met
-cod_met = lambda x: 5 <= x <= 15
+for object in response["Contents"]:
+    if re.search("csv", object["Key"]):
+        print("Deleting", object["Key"])
+        s3_client.delete_object(Bucket=s3_bucket, Key=object["Key"])
 
-prd_met = lambda x: 0.98 <= x <= 1.03
-
-prb_met = lambda x: -0.05 <= x <= 0.05
 
 # General boostrapping function
-def boot_ci(fun, *args, nboot = 100, alpha = 0.05):
-
+def boot_ci(fun, *args, nboot=100, alpha=0.05):
     num_args = len(args)
     args = pd.DataFrame(args).T
     n = len(args)
 
     ests = []
 
     for i in list(range(1, nboot)):
-        sample = args.sample(n = n, replace = True)
-        if fun.__name__ == 'cod' or num_args == 1:
+        sample = args.sample(n=n, replace=True)
+        if fun.__name__ == "cod" or num_args == 1:
             ests.append(fun(sample.iloc[:, 0]))
-        elif fun.__name__ in ['prd', 'prb']:
+        elif fun.__name__ in ["prd", "prb"]:
             ests.append(fun(sample.iloc[:, 0], sample.iloc[:, 1]))
 
     ests = pd.Series(ests)
 
     ci = [ests.quantile(alpha / 2), ests.quantile(1 - alpha / 2)]
 
-    ci = ', '.join([str(element) for element in ci])
+    ci = ", ".join([str(element) for element in ci])
 
     return ci
 
-# Formula specific bootstrapping functions
-def cod_boot(ratio, nboot = 100, alpha = 0.05):
 
-    return boot_ci(cod, ratio, nboot = nboot, alpha = alpha)
+# Formula specific bootstrapping functions
+def cod_boot(ratio, nboot=100, alpha=0.05):
```

Comment:
**[Question, non-blocking]** You mentioned in the PR body that this function is defined in `assesspy`, but isn't available for use by package consumers:

> things like cod_boot are included in the assesspy package, but can't be accessed from the package to use outside of it.

 Can you point me to the lines in `assesspy` where it's defined? I couldn't find it, but if it's true that it's defined in `assesspy`, we should try to figure out how to make it available to this script too.